src/System.Drawing.Primitives/src/System/Drawing/KnownColor.cs

```diff
@@ -45,9 +46,11 @@ public enum KnownColor
         Window,
         WindowFrame,
         WindowText,
+        */
```

Comment:
I decided to just keep them here so there is some reference to them from other platforms if we ever compare them.